portfolio/src/main/webapp/index.html

```diff
@@ -74,6 +74,27 @@ <h3>Connect with me on:</h3>
       <button type='button' onclick="addRandomQuote()">Quote</button>
       <div id="quote-container"></div>
 
+      <p>Click here to see my favorite TV shows:</p>
```

Comment:
Looking at the website, it seems like there's a lot of cool features. Perhaps it would be more readable if there were headings used to separate some of these items.

Reply:
Done